R/utility.R

```diff
@@ -556,11 +567,13 @@ waitForJobPreparation <- function(jobId, poolId) {
     # Verify that all the job preparation tasks are not failing
     if (all(FALSE %in% statuses)) {
       cat("\n")
-      stop(paste(
-        sprintf("Job '%s' unable to install packages.", jobId),
-        "Use the 'getJobFile' function to get more information about",
-        "job package installation."
-      ))
+      stop(
+        paste(
+          sprintf("Job '%s' unable to install packages.", jobId),
+          "Use the 'getJobFile' function to get more information about",
```

Comment:
can we do getJobFile on job packages? Is the api different?

Reply:
To be worked on later